sycl/include/sycl/ext/intel/experimental/esimd/memory.hpp

```diff
@@ -303,6 +293,33 @@ constexpr void check_lsc_atomic() {
   }
 }
 
+template <cache_hint L1H = cache_hint::none, cache_hint L3H = cache_hint::none>
+constexpr uint32_t get_lsc_cache_mask() {
```

Comment:
This function encodes load cache hints. Please implement similar function for store hints